zinit.zsh

```diff
@@ -77,6 +77,7 @@ pullopts|debug|null|binary|make|nocompile|notify|reset"
 typeset -g ZPFX
 : ${ZPFX:=${ZINIT[HOME_DIR]}/polaris}
 : ${ZINIT[ALIASES_OPT]::=${${options[aliases]:#off}:+1}}
+: ${ZINIT[MAN_DIR]:=${ZPFX}/man/man1}
```

Comment:
what about other man dirs? `man{1...9}` are all valid man directories. See `man man.1` for more info.

Reply:
Fair point. I don't have anything in my current config that uses anything other than man1. But yeah, valid. Will adjust.

Reply:
Done da91b76